improver/utilities/cube_manipulation.py

```diff
@@ -63,6 +63,11 @@ def collapsed(cube, *args, **kwargs):
     return new_cube
 
 
+def get_cube_dimensions(cube):
```

Comment:
Perhaps `get_dim_coord_names` would be a better name? I might interpret `get_cube_dimensions` as returning something like cube.shape, which isn't the intention. What do you think?

Reply:
Good idea.